packages/frontend/src/pages/ViewSqlChart.tsx

```diff
@@ -114,6 +114,7 @@ const ViewSqlChart = () => {
         projectUuid,
         slug: params.slug,
         limit: sqlChart?.limit,
+        additionalQueryKey: [params.slug, sql],
```

Comment:
This ensures that every time we create a sql chart, run a query, then change it (without running) then save, it won't get the cached results.